### ISSUE TITLE
How to Meet WCAG documents lacks mappings to g217 and F99 for SC 2.1.4

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1249]
These techniques should be added to the How to Meet document.

G217:   Providing a mechanism to allow users to remap or turn off character key   shortcuts
--
F99: Failure of Success Criterion   2.1.4 due to implementing character key shortcuts that cannot be turned off   or remapped

**Comment by alastc** [comment]
Hi Jon, another syncing issue.
